tools/docker/docker-compose.yml

```diff
@@ -1,8 +1,21 @@
 version: '3.9'
 
 services:
-  app:
-    image: ansible-events-ui
+  frontend:
+    image: eda-frontend
+    container_name: eda-frontend
+    build:
+      context: ../../
+      dockerfile: tools/docker/nginx/Dockerfile
+    restart: unless-stopped
+    ports:
+      - "80:80"
```

Comment:
This is incompatible with non-root setups. A regular user needs root privileges to use a port under 1024
It must be a port >1024

Reply:
Aah yes you are correct, I can switch this back and fix the readme to follow. I was thinking that in dev env most would be super user.